uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -1,123 +1,127 @@
+                <span>
+                  <i class="fa fa-envelope-o"></i>
+                </span> 
+                Send Welcome Email
+              </button>
+            </div>
+            <div class="crm_contact-actions__bottom-margin">
+              <button class="btn btn-primary-outline">
+                <span>
+                  <i class="fa fa-envelope-o"></i>
+                </span> 
+                Send Password Reset Email
+              </button>
+            </div>
+            <h3>User Has Access To:</h3>
+            <p>
+              <span class="crm_contact-actions__grey-text">No staff members</span> 
+              <span class="fa fa-question-circle crm_contact-actions__help_icon"></span>
+            </p> 
+            <button class="btn btn-secondary">Manage Regional Access</button>
           </div>
-          <h3>User Has Access To:</h3>
-          <p>
-            <span class="crm_contact_action_menu__grey_text">No staff members</span> 
-            <span class="fa fa-question-circle crm_contact_action_menu__help_icon"></span>
-          </p> 
-          <button class="btn btn-secondary-outline">Manage Regional Access</button>
         </div>
-        <div class="row crm_contact_action_menu__control_buttons_container">
-          <hr>
+        <div class="row crm_contact-actions__control-buttons-container">
+          <div class="col-md-12"><hr></div>
           <div class="col-md-6">
-            <button class="button__delete btn-warning">Disable User Account</button>
+            <button class="btn btn-warning">Disable User Account</button>
           </div>
           <div class="col-md-6">
-            <button class="button__disable btn-danger" 
+            <button class="btn btn-danger" 
               data-delete-user-url="civicrm/contactactionsmenu/deleteuseraccount">
               Delete User Account
             </button>
           </div>
         </div>
       </div>
-      <div class="col-md-8 crm_contact_action_menu__body">
+      <div class="col-md-8 crm_contact-actions__body">
         <div class="row">
           <div class="col-md-4">
             <h3>Leave:</h3>
-            <div class="crm_contact_action_menu__bottom_margin">
+            <div class="crm_contact-actions__bottom-margin">
               <button class="btn btn-primary-outline">
                 <span><i class="fa fa-plus"></i></span> Record New
               </button>
             </div>
-            <div class="crm_contact_action_menu__bottom_margin">
+            <div class="crm_contact-actions__bottom-margin">
               <button class="btn btn-primary-outline">
                 <span><i class="fa fa-search"></i></span> View Entitlements
               </button>
             </div>
             <hr>
-            <p><span class="crm_contact_action_menu__bold_text">Leave Approver(s):</span> </p>
+            <p><span class="crm_contact-actions__bold-text">Leave Approver(s):</span> </p>
```

Comment:
I still see so many `p` tags, didnot we decide we remove them?

Reply:
I have removed from all other parts. And I decided to keep `p` here as these are texts only.